test/src/097-statfs/setup_teardown

```diff
@@ -0,0 +1,21 @@
+test097_mount() {
+  local test097_mount=$1
+  local repo=$2
+  local params=$3
+
+  cvmfs_mount $repo $params
+}
+
+test097_unmount() {
+  cvmfs_umount $repo $params
```

Comment:
I think `$repo` is unknown here (may need `local repo=$1`) and we probably don't need `$params`